tests/ovs_test.go

```diff
@@ -32,4 +35,54 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		bridgeName := "br-test"
+		Context("When creating ovs-bridge on the host node", func() {
+			BeforeEach( func() {
+				err := removeOvsBridgeOnNode(bridgeName)
```

Comment:
I don't quite remember where we are with this, but I **do** remember a strong push from @EdDev to stop removing stuff in beforeEach's - each test should make sure whatever it created is removed. 

Since you're pretty much starting this module from scratch, I think that would be a good principle to use (if you have leaked stuff, let's not place a rug on top of it).

Reply:
I get what you mean. The purpose of adding this for the scenario where I stop the test manually so that the `AfterEach` is skipped, and so we can be left the ovs-bridge for the next test.
But I agree it's not enough of a reason, and it could put things under the rug. will remove.
Done